packages/fast-usdc/src/exos/status-manager.js

```diff
@@ -122,49 +171,81 @@ export const prepareStatusManager = zone => {
       },
 
       /**
-       * Find an `ADVANCED` or `OBSERVED` tx waiting to be `SETTLED`
+       * @param {CctpTxEvidence} evidence
+       */
+      isSeen(evidence) {
```

Comment:
`is` connotes a check on the current state but Seen isn't one of the states.

`hasBeenObserved`?